### PR TITLE
Added Thai missing translations

### LINKED ISSUE #38767: Missing translations for Thai (th)
Hello,

There are some missing translation for Thai (th). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.th.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.th.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.th.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.th.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.th.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.th.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by ibotdotout**
| Q             | A
| ------------- | ---
| Branch?       |  4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #38767
| License       | MIT
| Doc PR        | -

Added missing Thai translations based on 4.4 branch

CC. @Nyholm

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                <source>This value is not a valid HTML5 color.</source>
+                <target>ค่านี้ไม่ใช่ค่าที่ถูกต้องของค่าสี HTML5</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>กรุณากรอกวันเดือนปีเกิดที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>ตัวเลือกที่เลิอกไม่ถูกต้อง</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.th.xlf:27]: Can you double check if this is correct?

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                <source>The collection is invalid.</source>
+                <target>คอเล็กชั่นไม่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>กรุณาเลือกค่าสีที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>กรุณาเลือกประเทศที่ถูกต้อง</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.th.xlf:39]: Does this say "valid" or "correct"?

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>กรุุณาเลิอกค่าสกุลเงินที่ถูกต้อง</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.th.xlf:43]: Same ehre

**Review by Nyholm — COMMENTED** [review]
Awesome. Thank you. 

I used google translate to try to confirm these. I might be super wrong, but could you verify my comments?

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf (hunk 2 above)
Comment (ibotdotout) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:39]: It used the same term in Thai. 

You can double check by use Google Translate "Valid" to Thai then swap result to English again.
Valid -> ถูกต้อง, ถูกต้อง -> Correct

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf (hunk 3 above)
Comment (ibotdotout) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:43]: Google Trans Result: โปรดเลือกสกุลเงินที่ถูกต้อง

However, It same meaning anyway.

โปรด = กรุณา
สกุลเงิน = ค่าสุกลเงิน (ค่า = value)

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf (hunk 2 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:39]: Cool cool. Im just making sure.

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf (hunk 1 above)
Comment (ibotdotout) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:27]: It got the result "ตัวเลือกที่เลือกไม่ถูกต้อง" from Google Translate.
It same as I translated.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:27]: Google translate might be wrong. I just wanted you to confirm that this was correct. Thank you

File: src/Symfony/Component/Form/Resources/translations/validators.th.xlf (hunk 3 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.th.xlf:43]: Okey. Cool cool cool

**Review by Nyholm — APPROVED** [review]
Im happy with this :+1:

Thank you

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ibotdotout.

**Pull request merged**